### DISCUSSION TITLE
What's the difference of "Discussions" with "Issues" in Apache Doris?

### DISCUSSION THREAD

**Discussion by Myasuka** [apache/doris#6882] [RFC]
Is "Discussions" targeted for replacing previous Gitter? If so, shall we add documentation under http://doris.apache.org/master/zh-CN/community?

Moreover, after started voting in mail list to enable discussion, shall we add some descriptions to distinguish "Discussions" with "Issues"?

**Comment by morningman** [comment]
In my opinion, "Discussions" is more like a forum, mainly to help users with questions. "Issue" is more suitable for tracking bugs or features in development.
Of course, the two currently have some overlaps. We can try to guide users and developers to use these two sections on the official website.

If you have any suggestions, welcome to discuss.

**Reply by Myasuka** [reply]
Yes, I think we should better add guides in official site and could also have commented guides, just as what you did if creating a new PR, when creating any new disscussion.

Moreover, what's the rule of language to use in this discussion forum? Someone uses English while more guys prefer to use Chinese. 
From my experience of Flink community, we introduce a chinese only user mailling list for user discussion. However, Chinese users might not be familr with communicating via emails, we often found that they send emails with unseen pictures or bad format. Thus, I think github discussion is a good place to go. However, I think current Doris community might lack guide of how to create topic in discussions.

BTW, I have another question of why not using JIRA to track issues instead of github issues? That would be more better to track with a specific JIRA issue id.
